### ISSUE TITLE
WindowsFormsApplicationBase API

### ISSUE THREAD

**Issue by Royosef** [CXuesong/Shims.VisualBasic#1]
Hi there, I really don't know what is the right format for this kind of requests, but I would be very grateful if you could cover the `Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase` API as well. 

Thank You very much!

**Comment by CXuesong** [comment]
Wait… Suddenly I find out `My` namespace is back on .NET 5: https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.applicationservices.windowsformsapplicationbase?view=net-5.0

You can double check on this if upgrading SDK version is possible.

**Comment by Royosef** [comment]
Hey, you're absolutely right! It does work now, thank you!

(P.S. You made me retry to migrate to .Net 5 again (after previous failure), and this time I have  I have succeeded. So thanks for that too :D )

**Comment by CXuesong** [comment]
So I suppose there could have been a rally for bringing back the `My` namespace in .NET 5… (Well perhaps not: dotnet/runtime#26866)

At least there is no such namespace in .NET Core 3 and early previews of .NET 5.

**Comment by Royosef** [comment]
> Well perhaps not: dotnet/runtime#26866

Yeah, it seems you're right. . but I guess there is still be usages for that in the mean time.
